webapp/app/assessment/components/dataTable/table/cell/cellOdp.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { useSelector } from 'react-redux'
+
+import { isPrintingMode } from '@webapp/app/assessment/components/print/printAssessment'
+import { formatNumber } from '@common/bignumberUtils'
+// import { acceptNextDecimal } from '@webapp/utils/numberInput'
```

Comment:
Remove comments?

Reply:
It's a work in progress... Updates are not working, and I need that code. Commented out just for eslint